### DISCUSSION TITLE
Welcome to quagga2 Discussions!

### DISCUSSION THREAD

**Discussion by ericblade** [ericblade/quagga2#291] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ericblade** [comment]
Default discussion template above, because I didn't want to spend time to think about it :-)

Hello!  I'm Eric, and i'm presently maintaining and learning this repo.  We're in a very long term project that involves modernizing the repo, it's dependencies, and code, as well as a long term conversion of much of it into TypeScript.  Along the way, we've also significantly enhanced it's automated testing infrastructure, and squashed a lot of problems.  

Please feel free to open any issues or discussions :-)  Enjoy!

**Reply by emmanuelprince66** [reply]
thanks @ericblade  keep up the good work!
